timescaledb.rb

```diff
@@ -39,6 +40,12 @@ def install
     (share/"timescaledb").install Dir["stage/**/share/postgresql*/extension/*"]
   end
 
+  def check_postgresql_version
+    if postgresql.version >= Version.new('17.0') && postgresql.revision == 1
```

Comment:
```suggestion
    if postgresql.version >= Version.new('17.0') && postgresql.revision < 2
```
I think you should also add a check to avoid other minor versions that break timescaledb: 16.5, 15.9, 14.14, 13.17, and 12.21.